### ISSUE TITLE
Building on windows

### ISSUE THREAD

**Issue by kovidgoyal** [dottedmag/pychm#7]
I needed to change setup.py as follows to get it to build on windows

```
libraries=["chmlib"]
include_dirs=["/path/to/chmlib/include"],
library_dirs=["/path/to/chmlib.lib"],
define_macros=[("strcasecmp", "_stricmp"),("strncasecmp", "_strnicmp")]
extra_link_args=["/NODEFAULTLIB:MSVCRT"]
```

No idea if it will actually work, that testing will come later.

**Comment by kovidgoyal** [comment]
And importing the compiled module fails with:

```
python.exe -c "import os; from chm import _chmlib"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
SystemError: bad call flags
```

**Comment by kovidgoyal** [comment]
And the bad call flags bug is because you are missing the sentinel `{NULL}` in `chmlib_methods[]`. I guess it happened to work on linux by luck.

**Comment by kovidgoyal** [comment]
I suggest making a new release with the sentinel added.

**Comment by dottedmag** [comment]
Thanks, will be done ASAP.

**Comment by dottedmag** [comment]
I tested it on macOS, and there the memory was zero-filled. Bummer. Filed #8 to prevent such mishaps.

**Comment by dottedmag** [comment]
Released in 0.8.6
